pkg/configuration/load_configuration.go

```diff
@@ -0,0 +1,117 @@
+package configuration
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	errs "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+// LoadFromSecret retrieves an operator secret and sets a map
+// in order to override default configurations.
+// If no secret is found, then configuration will use defaults.
+// Returns error if WATCH_NAMESPACE is not set, if the resource GET request failed
+// (for other reasons apart from isNotFound) and if setting env vars fails.
+//
+// prefix: represents the operator prefix (HOST_OPERATOR/MEMBER_OPERATOR)
```

Comment:
the prefix is not used
```suggestion
```

Reply:
fixed here: bf4fb50b7964b03d59e810a8275ee5af586da8ac